### ISSUE TITLE
Use `McmcOptions` seed settings non-JAGS `mcmc` methods

### ISSUE THREAD

**Issue by wwojciech** [Roche/crmPack#190]
In some methods, e.g.
```
setMethod("mcmc",
    signature = signature(
        data = "Data",
        model = "LogisticIndepBeta",
        options = "McmcOptions"
    ),
    ...
)
```
we use random sample generator other than JAGS, e.g. `mvtnorm::rmvnorm`. This sample generator does not take into account seed settings (`rng_kind`, `rng_seed`) in `options` of `McmcOptions` class. It should be decided how we influence R sample generator to use seed setting from `options` of `McmcOptions` class.

**Comment by danielinteractive** [comment]
@wwojciech yeah I guess it would be consistent to also use the settings in these cases.
